package.json

```diff
@@ -24,11 +24,9 @@
     "labels:sync": "github-labels sync",
     "test": "jest --config jest.test.config.js",
     "test:watch": "yarn test --watch",
-    "test:e2e": "yarn test:e2e:docs-smoke-test && yarn test:e2e:api-docs-smoke-test",
```

Comment:
Why can't we have a script that runs all e2e tests for the three microsites? 🤔

Reply:
It's not used by automation and I never used it, but I can definitely leave it